app/views/authors/_toc_node.html.haml

```diff
@@ -57,8 +57,7 @@
     - collection = toc_node.collection
     - type_prefix = collection.collection_type == 'uncollected' ? '' : textify_collection_type(collection.collection_type)
     - uncollected = collection.collection_type == 'uncollected'
-    - any_published = collection.any_published_manifestations?
```

Comment:
this variable was not used, but it caused additional SQL query, so I've removed it